src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
That's a good idea, definitely using composition would be great.

I wonder if we should implement the wrappers with the existing names, and possibly come up with new names for the components (not necessarily easy) or export them off a new topic level object, eg

```
import { Controls } from '@jenkins-cd/design-lanaguage`;
const { Checkbox } = Controls;
```